tests/robots/test_robot.py

```diff
@@ -13,54 +18,160 @@
 
 @pytest.fixture
 def panda_srdf():
-    return os.path.join(BASE_FOLDER, 'fixtures', 'panda_semantics.srdf')
```

Comment:
not sure if this is really documented, but we (tend to) use single quote for strings. I'm sure we're not 100% consistent, but in general, that's the direction.

Reply:
I'll flip that back then!